scripts/follow_pr/wait-for-pr-activity.sh

```
@@ -0,0 +1,99 @@
+if [[ "$INPUT1" == http*://*github.com* ]]; then
+  # URL form: https://github.com/OWNER/REPO/pull/PR
+  if [[ "$INPUT1" =~ github.com/([^/]+)/([^/]+)/pull/([0-9]+) ]]; then
+    OWNER_REPO="${BASH_REMATCH[1]}/${BASH_REMATCH[2]}"
+    PR_NUMBER="${BASH_REMATCH[3]}"
+  else
+    echo "Could not parse PR URL: $INPUT1" >&2
+    exit 64
+  fi
+  SHIFTED=2
+elif [[ "$INPUT1" =~ ^([^/]+)/([^/]+)$ ]]; then
+  OWNER_REPO="$BASH_REMATCH[1]/${BASH_REMATCH[2]}"
```

Shellcheck (0.11.0)

[error] Use braces when expanding arrays, e.g. ${array[idx]} (or ${var}[.. to quiet).

(SC1087)